install.php

```diff
@@ -207,6 +250,186 @@ public function installOrUpdate($parent)
 	}
 
 	/**
+	 * Method to process SQL updates previous to the install process
+	 *
+	 * @param   object  $parent  Class calling this method
+	 *
+	 * @return  boolean          True on success
+	 */
+	public function preprocessUpdates($parent)
```

Comment:
why use this function at all? shouldn't Joomla run updater itself and run splitSql function?